index.bs

```diff
@@ -360,10 +360,12 @@ The presentation times of AV1 samples are given by the ISOBMFF structures. <asse
 
 <assert>The CleanApertureBox 'clap' SHOULD not be present.</assert>
 
-<assert>For sample entries corresponding to HDR content, the MasteringDisplayColourVolumeBox 'mdcv' and ContentLightLevelBox 'clli' SHOULD be present</assert>, and <assert>their values SHALL match the values contained in the [=Metadata OBUs=] of type METADATA_TYPE_HDR_CLL and METADATA_TYPE_HDR_MDCV, if present (in the configOBUs or in the samples)</assert>.
+<assert>For sample entries corresponding to HDR content, the MasteringDisplayColourVolumeBox 'mdcv' and ContentLightLevelBox 'clli' SHOULD be present</assert>, and <assert>their values SHALL equal (except for precision issues, see the second note below) the values contained in the [=Metadata OBUs=] of type METADATA_TYPE_HDR_MDCV and METADATA_TYPE_HDR_CLL, if present (in the configOBUs or in the samples)</assert>.
```

Comment:
Note that I exchanged METADATA_TYPE_HDR_CLL and METADATA_TYPE_HDR_MDCV here, so that they correspond to 'mdcv' and 'clli' in the right order.